metacat-connector-hive/build.gradle

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+dependencies {
+    compile project(':metacat-common-server')
+    compile project(':metacat-common')
+    compile "org.apache.hive:hive-metastore:1.2.1"
+    compile "org.apache.hadoop:hadoop-core:1.2.1"
+    compile "com.google.guava:guava:${guava_version}"
+    compile "com.google.inject:guice:${guice_version}"
+    compile "com.google.inject.extensions:guice-persist:${guice_version}"
+    compile "com.google.inject.extensions:guice-multibindings:${guice_version}"
+    compile "com.google.inject.extensions:guice-servlet:${guice_version}"
```

Comment:
All Guice dependencies are already likely in Metacat-Server-Common but if they're not you really don't need the servlet one here as you're not dealing with any web apis